blockchain/blockchain-wallet-v4-frontend
# feat(plugin): added switch accounts component

## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

![switch-account](https://user-images.githubusercontent.com/22176544/177121669-b86fcf80-b2d1-41be-880b-7212a1f860ac.gif)
